src/fl.go

```diff
@@ -129,14 +161,52 @@ func main() {
 	}
 
 	// Check if the "output" field exists
-	if verbose { fmt.Println("Checking AI output field...") }
+	verbosePrint("Checking AI output field...")
+	
 	result, ok := data["output"].(string)
 	if !ok {
 		fmt.Println("Error: Expected output field not found in Flows API response")
 		os.Exit(1)
 	}
-
+	
 	// Emit the result
-	if verbose { fmt.Println("Output: \n") }
+	verbosePrint("Output: \n")
+
 	fmt.Println(result)
+	fmt.Println()
+
+	// if not skipping prompt, ask user if they would like to execute
+	userExecute := false
+	if !noexec && !autoexecute {
+		var userInput string
+		fmt.Print("Would you like to execute the command? (y/n): ")
+		fmt.Scanln(&userInput)
+		userInput = strings.ToLower(userInput)
+		if userInput == "y" || userInput == "yes" {
+			userExecute = true
+		}
+	}
+
+	// perform the command if autoexecute enabled or user prompted to exec
+	if autoexecute || userExecute {
+		// convert to arr of values (exec requires a specific format)
+		fullCmd := strings.Split(result, " ")
+		cmd := fullCmd[0]
+		args := []string{}
+
+		if len(fullCmd) > 1 {
+			args=fullCmd[1:]
+		}
+
+		verbosePrint("Executing the result...")
+
+		out, err := exec.Command(cmd, args...).Output()
```

Comment:
this will work for single line returns from AI - you'll need to look at multi line returns and deal with those accordingly as multi execs potentially